packages/usa-input-prefix-suffix/src/usa-input-prefix.twig

```diff
@@ -22,7 +22,7 @@
   <div class="usa-input-group usa-input-group--error">
     <div class="usa-input-prefix" aria-hidden="true">
       <svg aria-hidden="true" role="img" focusable="false" class="usa-icon">
-        <use xmlns:xlink="http://www.w3.org/1999/xlink" href="./img/sprite.svg#credit_card"></use>
+        <use href="./img/sprite.svg#credit_card"></use>
       </svg>
```

Comment:
**Recommendation:** Reading the MDN docs for the SVG element, I believe we'll still need the `xmlns` attribute on the outer `svg` element.

> The xmlns attribute is only required on the outermost svg element of SVG documents, or inside HTML documents with XML serialization. [^1]

[^1]: https://arc.net/l/quote/jwgocjnj

So I believe we'd want it to look something like this:

```diff
+ <svg aria-hidden="true" role="img" focusable="false" class="usa-icon" xmlns="http://www.w3.org/2000/svg">
    <use href="./img/sprite.svg#credit_card"></use>
  </svg>
```

Open to other recommendations as well!

Reply:
@mahoneycm it could still be useful on the individual SVGs in case users want to pull those instead of the sprite. Smaller use case, but ensures the SVGs work everywhere.

Reply:
Resolved in [c9ab8a3](https://github.com/uswds/uswds/pull/6165/commits/c9ab8a3a0ae355e585e3c295211b2263946ccb78) and [1987f31](https://github.com/uswds/uswds/pull/6165/commits/1987f310553112293424288b29184b0f11d3e750).

@mejiaj Not sure I understand the "individual SVGs" comment? Do you mean adding it to the `<use>` element as well? Or is the link I've added for the `xlmns` on all the SVGs accomplishing that?

Reply:
Resolved in [ad97b5b](https://github.com/uswds/uswds/pull/6165/commits/ad97b5b5627d04772fce9726b12047bb08146f1c)